front_end/src/components/landing_page/search_widget/SearchWidget.js

```diff
@@ -78,6 +82,27 @@ class SearchWidget extends Component {
       this.props.handleResetSearchOverview();
     }
 
+    //Check if the destination is given or not.
+    if (this.state.destinationName === "") {
+      this.setState({ showDesAlert: true });
+    } else {
+      this.setState({ showDesAlert: false });
+    }
+    //Check if the checkIn and checkOut date are given or not.
+    if (this.state.checkIn === "" || this.state.checkOut === "") {
```

Comment:
checkIn and checkOut can be null if a user enters a date and then manually deletes it. Please also check for null.